cms/cms-import.md

```diff
@@ -0,0 +1,275 @@
+
+```yml
+imports:
+  import-name-random: 
+    identifier: 
+```
+<a name='handle_duplicated_identifier'></a>
+
+#### `handle_duplicated_identifier`
+The way to handle duplicated records when If more than 1 record in the database is found to have the same identifier as the imported record.
+
+```yml
+imports:
+  import-name-random: 
+    handle_duplicated_identifier: throw-error or use-first
+```
+
+- `throw-error` will return an error about this record. If you want to reject the whole import when there are errors, you can set `atomic: true` in the `action` setting (will be explained below).
+
+- `use-first` will randomly accept one of the duplicated records.
+
+<a name='fields'></a>
+
+#### `fields`
+Fields the users should fill in. 
+
+For relational records, Import support only **reference** (i.e. 1-many).
+
+```yml
+imports:
+  import-name-random: 
+    fields:
+      - name: a normal field
+      - name: a reference field
+        reference_target: the referenced table 
+        reference_field_name: the field user should fill in the CSV to idenitfy the referenced record
+        handle_duplicated_reference: throw-error or use-first
+```
+- `throw-error` will return an error. If you want to reject the whole import when there are errors, you can set `atomic: true` in the `action` setting (will be explained below).
+
+- `use-first` will randomly assign the record to one of the duplicated references.
+
+### Add the import button
+
+To add an import button to the page, you need configure it in the list view. 
+
+Example usage:
+
+Suppose now we are adding an **import** button on the `comment`'s list view.
+
+```yml
+site:
+  - type: Record
+    name: comment
+
+records:
+  comment:
+    record_type: comment
+    list:
+      actions:
+        - type: Import
+          name: import-comment # this is defined in the import section
+          label: Import
+          atomic: true 
+```
+
+- `atomic`: When `atomtic` is set to true, the whole import will be rejected if there are errors from `handle_duplicated_reference` and `handle_duplicated_identifier`. Default is false.
+
+## Exports
+
+![Skygear CMS export](/assets/cms/cms-export.png)
+
+Your users can export data from Skygear to a CSV. To do so, you need to configure the export settings and add an export button to the right page.
+
+### Configuring exports
+
+Reusing the two databases in the Configuring imports section as an example.
+
+This is what we need:
+
+1. Allow users to export all the comments
+2. Next to each comment, we will also show the blogpost title the comment belongs to
+
+```yml
+site:
+  - type: Record
+    name: comment
+    label: Comment
+
+records:
+  comment: 
+    record_type: comment
+  blogpost:
+    record_type: blogpost
+
+exports:
+  export-comments:
+    record_type: comment
+    fields:
+      - name: _id
+      - name: _created_at
+      - name: comment
+      - name: blogpost_title
```

Comment:
`label` can be specified, which is used in the the header row.

Reply:
> `label` can be specified, which is used in the the header row.

Does it apply to both imports and exports?

Reply:
It will be supported for import, not implemented yet.

For import, it is for matching which column is which field of the record.

Reply:
Thanks. Fixed.